components/common/itemTransfer/ItemTransferModal.vue

```diff
@@ -0,0 +1,200 @@
+            </p>
+          </div>
+        </div>
+      </div>
+    </NeoModal>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoButton, NeoIcon, NeoModal } from '@kodadot1/brick'
+import ModalIdentityItem from '@/components/shared/ModalIdentityItem.vue'
+import { NFT } from '@/components/rmrk/service/scheme'
+import BasicImage from '@/components/shared/view/BasicImage.vue'
+import { parseNftAvatar } from '@/utils/nft'
+import AddressInput from '@/components/shared/AddressInput.vue'
+import { Interaction } from '@kodadot1/minimark/v1'
+import formatBalance from '@/utils/format/balance'
+import { decodeAddress, encodeAddress } from '@polkadot/util-crypto'
+
+const emit = defineEmits(['close'])
+const props = defineProps<{
+  nft: NFT
+  value: boolean
+}>()
+
+const { $route, $i18n } = useNuxtApp()
+const { transaction, status, isLoading } = useTransaction()
+const { urlPrefix } = usePrefix()
+const { decimals, chainSymbol, chainProperties } = useChain()
+const ss58Format = computed(() => chainProperties.value?.ss58Format)
+const { accountId } = useAuth()
+
+const isModalActive = useVModel(props, 'value')
+const avatar = ref<string>()
+const address = ref('')
+const isAddressValid = ref(false)
+
+const transferItemLabel = computed(() => {
+  if (!address.value) {
+    return $i18n.t('transaction.inputAddressFirst')
+  }
+  if (!isAddressValid.value) {
+    return $i18n.t('transaction.addressIncorrect')
+  }
+  if (isYourAddress.value) {
+    return $i18n.t('transaction.cantTransferToYourself')
+  }
+  return $i18n.t('transaction.transferNft')
+})
+
+const isYourAddress = computed(
+  () => accountId.value === getChainAddress(address.value)
+)
+
+const isDisabled = computed(
+  () => !address.value || !isAddressValid.value || isYourAddress.value
+)
+
+const nftPrice = computed(() =>
+  Number(props.nft.price)
+    ? formatBalance(Number(props.nft.price), decimals.value, chainSymbol.value)
+    : '--'
+)
+
+const getAvatar = async () => {
+  if (props.nft) {
+    avatar.value = await parseNftAvatar(props.nft)
+  }
+}
+
+const onClose = () => {
+  emit('close')
+}
+
+const handleAddressCheck = (isValid) => {
+  isAddressValid.value = isValid
+}
+
+const getChainAddress = (value: string) => {
+  if (!value) {
+    return null
+  }
+  const publicKey = decodeAddress(value)
+  return encodeAddress(publicKey, ss58Format.value)
+}
+
+const transfer = () => {
+  transaction({
+    interaction: Interaction.SEND,
+    urlPrefix: urlPrefix.value,
+    address: address.value,
+    tokenId: $route.params.id,
+    nftId: props.nft.id,
+    successMessage: $i18n.t('transaction.item.success') as string,
+    errorMessage: $i18n.t('transaction.item.error') as string,
+  })
+}
+
+onMounted(() => {
+  getAvatar()
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.modal-width {
+  width: 25rem;
+}
+
+@include touch() {
+  .modal-width {
+    width: unset;
+  }
+}
+.btn-height {
+  height: 3.5rem;
+}
```

Comment:
`.btn-height` already exist in modals.scss

Reply:
It's a bit smaller  height: 3.25rem; about 50px from what I've seen other modals have this height which is 56px

Reply:
can't this be unified?

Reply:
I dont mind using that class 

idk maybe cc @exezbcz do all action btn inside modals have  `height: 3.5rem;`  and some have `height: 3.25rem;` ?